app/src/main/java/com/klinker/android/twitter/activities/compose/Compose.java

```diff
@@ -763,16 +772,17 @@ private void startUcrop(Uri sourceUri) {
     public static final int PWICCER = 420;
 
     public boolean pwiccer = false;
+    private boolean multiTweets = true;
```

Comment:
Shouldn't we default this to `false`, then set it to `true`, if that is the option the user clicked on? Right now, it seems like there isn't a way to get to TwitLonger anymore, which used to be the default.

Since this is always `true`, it will always go through that code block, even if the user selected TwitLonger, instead of the multi-part tweets.